.github/workflows/reviewdog.yml

```diff
@@ -0,0 +1,36 @@
+name: Reviewdog
+
+on:
+  pull_request:
+    types: [opened, review_requested]
+
+jobs:
+  reviewdog:
+    runs-on: ubuntu-latest
+
+    if: github.event.action == 'opened' || github.event.requested_team.name == 'reviewdog'
```

Comment:
Although I couldn't find `requested_team` in the payload documentation, but it actually exists and it is documented in the actions reference.

- payload: https://docs.github.com/en/rest/reference/pulls#get-a-pull-request
- actions document: https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#pull_request